src/controller/file_manager.controller.ts

```diff
@@ -5,9 +5,15 @@ import { ApiBearerAuth, ApiResponse, ApiTags } from '@nestjs/swagger';
 import { Request } from '../shared/request';
 import { FileManagerService } from '../service/file_manager/file_manager.service';
 import { FileManagerFormatter } from '../service/file_manager/file_manager.formatter';
-import { FolderResponse, FileManagerListResponse, SignedPostUrlsResponse } from '../interface/apiResponse';
+import {
+  FolderResponse,
+  FileManagerListResponse,
+  SignedPostUrlsResponse,
+  SignedGetUrlsResponse,
+} from '../interface/apiResponse';
 import {
   CreateFolderRequest,
+  DownloadFileRequest,
```

Comment:
Can you please change naming from DownloadFileRequest to something related to urls. And the same for UploadFileRequest

Reply:
I'll change them to ReceiveUrlGetRequest & ReceiveUrlPostRequest then.